frontend/models/Tasks.php

```diff
@@ -126,4 +135,28 @@ public function getExecutor()
     {
         return $this->hasOne(Users::class, ['id' => 'user_id_executor']);
     }
+
+    /**
+     * Сохраняет файл в нужную директорию, присваивает имя файлу
+     * и сохраняет файл и id связной задачи в таблицу
+     *
+     * @param string $task_id id созданного задания
+     */
+    public function upload($task_id)
+    {
+        if ($files = UploadedFile::getInstances($this, 'clips')) {
+            foreach ($files as $file) {
+                $newname = $file->baseName . '.' . $file->getExtension();
```

Comment:
По-хорошему, нужно файлу генерировать случайное имя, т.к. сейчас есть возможность перезаписать файл в случае сопадения имён. Можно либо полностью генерировать уникальное имя, либо добавить к оригинальному имени:
```suggestion
                $newname = $file->baseName . '.' . uniqid('', true) . '.' . $file->getExtension();
```